src/Element/BaseInput.php

```
@@ -170,14 +172,22 @@ protected function loadDefault(): array
     private function renderTag(
-        BlockInterface|false|InlineInterface|VoidInterface $tag,
+        BackedEnum|false $tag,
         string $content,
         array $attributes = [],
     ): string {
         if ($tag === false) {
             return $content;
         }
 
+        if ($tag instanceof Voids || $tag instanceof MetadataVoid) {
+            return Html::void($tag, $attributes);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Html' in method 'renderTag'. (undefined)

(StaticAccess)